js/widgets/rhythmruler.js

```diff
@@ -1145,18 +1145,22 @@ function RhythmRuler () {
         // FIXME: rough workaround for #508, investigate reasons why
         // the backspace press doesn't work by default
         var numberInput = docById('dissectNumber');
-        numberInput.addEventListener('keydown', function (event) {
-            if (event.keyCode === BACKSPACE) {
-               numberInput.value = numberInput.value.substring(0, numberInput.value.length - 1);
-            }
-
+        numberInput.onkeydown =  function (event) {
+            if (event.keyCode == 46) {
```

Comment:
Would be good to define DEL as a const for 46.
So the prob. was we were treating backspace like delete?

Reply:
Yes .
I will make the change.